engine/src/main/java/com/datatorrent/stram/StramLocalCluster.java

```diff
@@ -318,6 +333,21 @@ public StramLocalCluster(LogicalPlan dag) throws IOException, ClassNotFoundExcep
     }
   }
 
+  private void addExtraJarsToClasspath(Set<String> jars)
+      throws InvocationTargetException, IllegalAccessException, MalformedURLException, NoSuchMethodException
+  {
+    List<URL> jarUrls = new LinkedList<>();
+    for (String jarPath : jars) {
+      File file = new File(jarPath);
+      URL url = file.toURI().toURL();
+      jarUrls.add(url);
+    }
+
+    ClassLoader prevCl = Thread.currentThread().getContextClassLoader();
+    ClassLoader urlCl = URLClassLoader.newInstance(jarUrls.toArray(new URL[jarUrls.size()]), prevCl);
+    Thread.currentThread().setContextClassLoader(urlCl);
```

Comment:
I doubt that this provides required functionality as for example Class.forName() will not work if it is called from the top level module or operator loaded by the system ClassLoader.